.hooks/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+stage_files=$(git diff --cached --name-only | grep "tests_mark_conditions*")
+if [[ -z $stage_files ]];then
+  exit 0
+fi
+chmod +x .hooks/pre-commit.py
+python .hooks/pre-commit.py $stage_files
+if [ $? != 0 ];then
+    echo "Pleace check the order in tests/common/plugins/conditional_mark/tests_mark_conditions*.yaml"
+    exit 1
+else
+    exit 0
+fi
```

Comment:
Usually we add a new line to the end of file. You can configure your VSC to automatically add it for you.

Reply:
done